plugins/inputs/t128_graphql/sample.go

```diff
@@ -13,7 +13,7 @@ var sampleConfig = `
 # unix_socket = "/var/run/128technology/web-server.sock"
 
 ## The starting point in the graphQL tree for all configured tags and fields
-# entry_point = "allRouters[name:RTR_WEST_COMBO]/nodes/nodes[name:combo-west]/nodes/arp/nodes"
+# entry_point = "allRouters(name:\"ComboEast\")/nodes/nodes(name:\"combo-east\")/nodes/arp/nodes"
```

Comment:
Doing the `\"` is not very intuitive. Why do you need the quotes now - they seem un-necessary? Can you use single quotes instead maybe?

Reply:
The single quotes is a better solution. I couldn't find the exact query you showed me but the predicate was something like `(metric: AVERAGE, key2: VALUE2)`. So the code needs to handle that case and the one in `sample.go` above where quotes are needed

Reply:
The only time we probably need quotes is when the keys have a space in them. For the sake of this config, I'd say we should support both with and without single quotes. If there are no single quotes needed then we handle the above scenario. If the input has a special character or space etc then expect a single quote right after the `:` - this way the config is more intuitive for the user